### ISSUE TITLE
make a package

### ISSUE THREAD

**Issue by ben-williams** [afsc-assessments/AK-projection-simulation#1]
seems this may lend itself to being a forecast package. would be easier to maintain timeline (version control) - thoughts?

**Comment by Cole-Monnahan-NOAA** [comment]
Agreed -- but only if people will use it. If everyone sticks to the command line then it's probably not worth it. I'd be willing to help convert and maintain moving forward if we go that way.

**Comment by Cole-Monnahan-NOAA** [comment]
Wow blast from the past. I'm leaning more toward a package because it allows for easier testing and distribution. Users may never install it but I think the structure would be helpful in general

**Comment by jimianelli** [comment]
So...revisiting this topic, some folks have automatic outputs generated from their assessments as inputs to the old proj so maybe the list for settings and pointing to input should be based on reading that (so e.g., a package with a function say "do_proj(mylist)" is a function that takes mylist as an ascii from previous inputs.

**Comment by BenWilliams-NOAA** [comment]
i started a "package" branch. stripped out all the non-package components (e.g., cannot have .tpl in src folder etc.) - it is a starting point for a functional package anyway

**Comment by BenWilliams-NOAA** [comment]
Oh yeah - I have a projection function we've been using the past few years nested  here https://github.com/BenWilliams-NOAA/afscassess/blob/main/R/proj.R
